whether-page/whether.js

```diff
@@ -0,0 +1,31 @@
+const apiKey = "";
+    const cityBox = document.querySelector(".city-name");
+    const weatherIcon = document.querySelector(".weather-icon");
+    async function checkwhether(city) {
+        const apiUrl = `https://api.openweathermap.org/data/2.5/weather?units=metric&q=${city}&appid=${apiKey}`;
+        const response = await fetch (apiUrl);
+        var data = await response.json();
+        document.querySelector(".city").innerHTML = data.name;
+        document.querySelector(".temp").innerHTML = Math.round(data.main.temp) + "°C";
+        document.querySelector(".humidity").innerHTML = data.main.humidity + "%";
+        if(data.weather[0].main=="Clouds"){
```

Comment:
If there are a lot of if else condition then we can prefer using switch case method it will help in code readablity.